app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= render partial: "shared/navbar", locals: { location: 'NEW TRANSACTION'} %>
+
+<div class="form_category">
+  <%= form_with model: @transaction, url: category_transactions_path  do |f| %>
+
+    <div class="form_inputs">
+      <div>
+        <%= f.text_field :name, placeholder: "Name", class: "input"%>
+      </div>
+
+      <div>
+        <%= f.number_field :amount, placeholder: "Price", class: "input"%>
+      </div>
+    </div>
```

Comment:
Required

- In the new transaction form view, please add a field with all the groups names to select from as assignment to this new transaction. This is to make use of the many to many relationship. Currently, the user can only add a transaction to one group, but the project requirements dictate that q user should be able to assign the same transaction to as many groups as wanted, therefore the ERD association will be applied. See the image from the project requirements down below.

![Screenshot_2022-08-13-18-59-19-68](https://user-images.githubusercontent.com/47625407/184504305-979a0155-008c-4044-b548-1d224620811b.jpg)